### ISSUE TITLE
Create Java code snippets for streams

### ISSUE THREAD

**Issue by sav-norem** [redis/jedis#3632]
The redis.io webpage for [streams](https://redis.io/docs/data-types/streams/) needs the Java example as well. The python equivalent can be used to copy the setup and can be found [here](https://github.com/redis/redis-py/blob/emb-examples/doctests/dt_stream.py)

The first line of the file needs to be the ID for the example - # EXAMPLE: stream_tutorial

Each separate section is delimited with a tag that you will need to copy exactly for it to appear in the correct place, ie.
```
# STEP_START xadd
{code}                                    
# STEP_END

# HIDE_START, # HIDE_END, # REMOVE_START, and # REMOVE_END can be used for debugging and deleting keys. Hide will hide until a user expands the example, remove will only be seen in the source code and will never appear on the webpage.
```

@dwdougherty and I will be reviewing and tracking these issues - feel free to tag us with any questions

**Comment by Harsh4902** [comment]
@sav-norem I want to work on this issue. Can you please tell me In which module I have to put code snippet?

**Comment by sav-norem** [comment]
@Harsh4902 it should live in this folder - https://github.com/redis/jedis/tree/master/src/test/java/io/redis/examples and you can see another example in there with the SetGetExample for what we're looking for! thanks!

**Comment by Harsh4902** [comment]
@sav-norem please review this.
https://github.com/redis/jedis/pull/3640

**Comment by dwdougherty** [comment]
I reviewed and commented.

@sav-norem You might want to give the PR a scan, since you know the examples better than I do.

**Comment by sazzad16** [comment]
Resolved by #3641
